.prettierrc

```diff
@@ -4,7 +4,8 @@
   "printWidth": 120,
   "trailingComma": "es5",
   "tabWidth": 2,
-  "importOrder": ["^@core/(.*)$", "^@server/(.*)$", "^@ui/(.*)$", "^[./]"],
+  "importOrder": ["^[./]"],
   "importOrderSeparation": true,
-  "importOrderSortSpecifiers": true
+  "importOrderSortSpecifiers": true,
+  "importOrderCaseInsensitive": true
```

Comment:
one question about case insensitive, maybe a nit: If we have a file naming convention designed to separate react components from utility / business logic files based on casing, are you sure you want to make the sorting case insensitive?

Reply:
So I tried implementing this by changing the sorting to separate lowercase and uppercase filenames, but it breaks down in a few places based on our naming conventions.


Good lowercase + uppercase sorting:
```
// utilities
import { getColorMap, getInRangeLUT, thresholdMatchFinder, validateThresholds } from "./colorizer/utils/data_utils";
import { numberToStringDecimal } from "./colorizer/utils/math_utils";

// classes
import Collection from "./colorizer/Collection";
import Export from "./components/Export";
```

Bad Lowercase sorting:
```
// utilities
import { ColorizeCanvas, Dataset, Track } from "./colorizer";
import { FeatureThresholdsTab, PlotTab, ScatterPlotTab, SettingsTab, TabType } from "./components/tabs";
```

Basically, this starts to break down anytime modules have been introduced. It raises the question of whether these module names should be capitalized if they mostly include business logic.

Reply:
(Oh and just to add, afaik the case insensitive sorting is only applied within imports per module.)

Reply:
Okay, added this and refactored a few submodule names. It's not perfect, but I do like the new separation between components and logic.